cloud/aws/templates/aws_oidc/bin/aws_cli.py

```diff
@@ -109,6 +109,23 @@ def wait_for_ecs_service_healthy(self):
             )
             time.sleep(30)
 
+    def fix_digest_value(self, value):
+        """
+        Sets the lock file digest value in DynamoDB to the given value.
+
+        If something goes wrong during deployment, especially when a user has
+        force-unlocked due to a previous issue and then multiple apply actions
+        are happening at once, the digest value for the Terraform lock file in
+        S3 can be incorrect. This function lets us set the digest value to
+        the correct value, as given by the error message of a previous
+        Terraform command, without having to go into the AWS console to
+        set it manually.
+        """
+        table = f'{self.config.app_prefix}-{resources.S3_TERRAFORM_LOCK_TABLE}'
+        file = f'{self.config.app_prefix}-{resources.S3_TERRAFORM_STATE_BUCKET}'
+        command = f'dynamodb put-item --table-name={table} --item=\'{{"LockID":{{"S":"{file}/tfstate/terraform.tfstate-md5"}},"Digest":{{"S":"{value}"}}}}\''
+        self._call_cli(command, False)
```

Comment:
**nitpick:** either add an inline comment hinting at what the boolean is or use an enum param instead of a boolean